locales/en/app.ftl

```diff
@@ -12,6 +12,7 @@
 -brand-lockwise = Firefox Lockwise
 -brand-send = Firefox Send
 -brand-fpn = Firefox Private Network
+-brand-fpn-2 = Mozilla VPN
```

Comment:
Why not `-brand-mozilla-vpn` at this point?

Reply:
Good call. That'll fix any confusions later too. Updating!